tests/providers/cncf/kubernetes/operators/test_kubernetes_pod.py

```diff
@@ -102,15 +106,14 @@ def run_pod(self, operator: KubernetesPodOperator, map_index: int = -1) -> k8s.V
         remote_pod_mock = MagicMock()
         remote_pod_mock.status.phase = 'Succeeded'
         self.await_pod_mock.return_value = remote_pod_mock
-        if not isinstance(self.hook_mock.return_value.is_in_cluster, bool):
-            self.hook_mock.return_value.is_in_cluster = True
```

Comment:
instead of doing this, i just read the value from the actual hook (since we're no longer patching it globally)